oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/rdb/RDBDocumentNodeStoreBuilder.java

```diff
@@ -128,6 +128,18 @@ public RDBDocumentNodeStoreBuilder setFullGCEnabled(boolean b) {
         return thisBuilder();
     }
 
+    @Override
+    public RDBDocumentNodeStoreBuilder setFullGCMode(int v) {
+        // fullGC modes are not supported for RDB
```

Comment:
should we then throw something like an `IllegalArgumentException` here?

Reply:
Since `fullGC` is supported for RDB, logging a `WARN` message should be sufficient.

Reply:
added log.warn, thank you for the suggestion.